src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -641,6 +691,47 @@ internal static string NormalizeFilePath(string path)
             return Path.GetFullPath(path);
         }
 
+        /// <summary>
+        /// Determine the execution policy based on the supplied string.
+        /// If the string doesn't match to any known execution policy, set it to incorrect.
+        /// </summary>
+        /// <param name="_executionPolicy">The value provided on the command line.</param>
+        /// <returns>The execution policy.</returns>
+        private static ParameterBitmap GetExecutionPolicy(string? _executionPolicy)
+        {
+            if (_executionPolicy is null)
+            {
+                return ParameterBitmap.EPUndefined;
+            }
+
+            if (String.Equals(_executionPolicy, "remotesigned", StringComparison.OrdinalIgnoreCase))
+            {
+                return ParameterBitmap.EPRemoteSigned;
+            }
+            else if (String.Equals(_executionPolicy, "bypass", StringComparison.OrdinalIgnoreCase))
```

Comment:
'else if' is not needed since we do an early return.

Reply:
turned the whole thing into a switch